media-gfx/xpaint/files/xpaint-2.10.2-respect-ldflags.patch

```diff
@@ -0,0 +1,15 @@
+--- a/util/Makefile	2017-03-14 20:54:48.000000000 +0200
```

Comment:
I'd drop the [timestamps](https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#clean-patch-howto) but it's not a big deal, more an FYI.